src/main/frontend/components/plugins.cljs

```diff
@@ -437,15 +437,15 @@
          [:option "https://s3.amazonaws.com"]
          [:option "https://clients3.google.com/generate_204"]]]
 
-       (ui/button (if testing? (ui/loading "Testing") "Test URL")
+       (ui/button (if testing? (ui/loading (t :network-proxy/test-load)) (t :network-proxy/test))
                   :intent "logseq" :large? false
                   :on-click #(let [val (util/trim-safe (.-value (rum/deref *test-input)))]
                                (when (and (not testing?) (not (string/blank? val)))
                                  (set-testing?! true)
                                  (-> (p/let [result (ipc/ipc :testProxyUrl val opts)]
                                        (js->clj result :keywordize-keys true))
                                      (p/then (fn [{:keys [code response-ms]}]
-                                               (notification/show! (str "Success! Status " code " in " response-ms "ms.") :success)))
+                                               (notification/show! (str (t :network-proxy/test-notif-1) code (t :network-proxy/test-notif-2) response-ms "ms.") :success)))
```

Comment:
We can't rely on word ordering to match English so it'd be good to pass code and response-ms as [an argument](https://github.com/logseq/logseq/blob/master/docs/contributing-to-translations.md#editing-tips). For example, `"Success! Status {1} in {2} ms."` with `(t :network-proxy/test-notification code response-ms)`